python/lsst/ts/observatory/control/maintel/mtcalsys.py

```diff
@@ -240,12 +207,105 @@ async def setup_calsys(self, sequence_name: str) -> None:
             await self.linearstage_projector_select.cmd_moveAbsolute.set_start(
                 distance=self.ls_select_laser_location, timeout=self.long_timeout
             )
-            await self.setup_laser(config_data["laser_mode"])
-            await self.rem.tunablelaser.cmd_startPropagating.start(
-                timeout=self.long_long_timeout
+            await self.setup_laser(
+                config_data["laser_mode"],
+                config_data["wavelength"],
+                config_data["optical_configuration"],
+            )
+            await self.laser_start_propagate()
+
+    def calculate_laser_focus_location(self, wavelength: float | None) -> float:
+        """Calculates the location of the linear stage that provides the focus
+        for the laser projector. This location is dependent on the
+        wavelength of the laser.
+
+        Parameters
+        ----------
+        wavelength : `float`
+            wavelength of the laser projector in nm
+
+        Returns
+        ------
+        location of the linear stage for the laser projector focus in mm
+
+        """
+        # TODO (DM-44772): implement the actual function
+        return 10.0
+
+    async def change_laser_wavelength(
+        self,
+        wavelength: float | None,
+        use_projector: typing.Optional[bool] = True,
```

Comment:
Since this is a bool value with a default, you don't actually need to make it Optional. You can just use `use_projector: bool = True`. The typing.Option[bool] is actually equivalent to the way you used e.g. `bool | None` above.